docs/components/code/highlight.global.scss

```diff
@@ -0,0 +1,198 @@
+div.hljs-filename {
```

Comment:
I assume this is all custom styles for the syntax highlighter?

Reply:
Why not use the syntax highlighter similarly to how we use it on the main app? [(e.g.)](https://github.com/dataform-co/dataform-co/blob/master/ts/app/components/code_viewers/code_view.tsx)

As in passing in styles?